### DISCUSSION TITLE
ip地址不在白名单

### DISCUSSION THREAD

**Discussion by MZTol** [BurnHair/docs#202] [分享讨论]
现在使用一直会出现ip地址不在白名单，换了好多个节点也这样。。。。。

**Comment by BennyThink** [comment]
IP地址不在白名单是什么意思，有图吗

**Reply by MZTol** [reply]
![image](https://github.com/user-attachments/assets/2fa9820c-db46-483a-8c67-cd11ba7be16f)

**Reply by MZTol** [reply]
就是这样，切换其他节点也会，其他设备也一样，都是这个message

**Reply by BennyThink** [reply]
你的令牌是不是设置了访问白名单？

**Reply by BennyThink** [reply]
你设置了白名单，要么取消设置，要么修改你的白名单
<img width="231" alt="image" src="https://github.com/user-attachments/assets/310e2eac-a832-4888-82aa-695aa2dab05f">

**Reply by MZTol** [reply]
还真是，草率了，我的我的，不知道那是啥东西就直接点了追加[捂脸]

**Reply by BennyThink** [reply]
解决了就好
